among-eus-frontend/src/components/MapOverview/MapOverview.css

```diff
@@ -44,10 +44,14 @@
     background-color: red;
 }
 
-.kill-action-button {
+.kill-banner {
```

Comment:
Name conflict - this style is not realy for the kill banner but for the kill notification

Reply:
better rename all that is for notfications to "notification" instead of "banner"